### PR TITLE
Updates for shortcode clipboard issue on PDF List (settings)

### PR DESCRIPTION
**Pull request by jestonihpi**
## Description
Based on the approved prototype https://jsfiddle.net/kxodnhr6/12/. 

Remove `autoSelectShortcode` and replace with `shortcodeButton`.
Added new import for `shortcodeButton`
Update stylesheet for the new shortcode button.

#1246 
## Checklist:
- [x] I've tested the code.
- [x] My code is easy to read, follow, and understand
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation / docblocks.

### CODE REVIEW

File: src/Helper/Helper_Data.php [hunk 1]
```diff
@@ -330,6 +330,10 @@ public function get_localised_script_data( Helper_Abstract_Options $options, Hel
 				'searchBoxPlaceHolderText'             => esc_html__( 'Search the Gravity PDF Documentation...', 'gravity-forms-pdf-extended' ),
 				'searchBoxSubmitTitle'                 => esc_html__( 'Submit your search query.', 'gravity-forms-pdf-extended' ),
 				'searchBoxResetTitle'                  => esc_html__( 'Clear your search query.', 'gravity-forms-pdf-extended' ),
+
+				/* Shortcode Button*/
+				'copyShortcode'                        => esc_html__( 'Copy Shortcode', 'gravity-forms-pdf-extended' ),
+				'shortcodeCopy'                        => esc_html__( 'Shortcode copied!', 'gravity-forms-pdf-extended' ),
```
Comment (jakejackson1) [inline src/Helper/Helper_Data.php]: I'd rename this `shortcodeCopied` instead.

File: src/Helper/Helper_PDF_List_Table.php [hunk 2]
```diff
@@ -281,15 +281,15 @@ public function column_shortcode( $item ) {
 		$pdf_id = esc_attr( $item['id'] );
 
 		/* Prepare our shortcode sample */
-		$shortcode = '[gravitypdf name="' . esc_attr( $name ) . '" id="' . esc_attr( $item['id'] ) . '" text="' . esc_attr__( 'Download PDF', 'gravity-forms-pdf-extended' ) . '"]';
+		$shortcode  = '[gravitypdf name="' . esc_attr( $name ) . '" id="' . esc_attr( $item['id'] ) . '" text="' . esc_attr__( 'Download PDF', 'gravity-forms-pdf-extended' ) . '"]';
 		$aria_label = sprintf( esc_html__( 'Select to copy the PDF %s shortcode to the clipboard', 'gravity-forms-pdf-extended' ), esc_html( $item['name'] ) );
```
Comment (jakejackson1) [inline src/Helper/Helper_PDF_List_Table.php]: We could probably remove the `esc_html` references here, as it gets run through `esc_attr()` below.

**Review by jakejackson1 — CHANGES_REQUESTED** [review]
Minor PHP adjustments, plus the JS linting errors need to be fixed.

File: src/Helper/Helper_PDF_List_Table.php (hunk 2 above)
Comment (jestonihpi) [reply src/Helper/Helper_PDF_List_Table.php]: Copy @jakejackson1 .

**Review by debbieellis7 — APPROVED** [review]

**Comment by jakejackson1** [comment]
@debbieellis7 @jestonihpi I refined this PR over the weekend and it needs another check over. Also, we need to add some JS tests / E2E tests before it gets merged in.

**Review by jakejackson1 — APPROVED** [review]

**Pull request merged**